<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{
border-image: url(:/image/2/1.png);
}</string>
   </property>
   <widget class="QLineEdit" name="usernameLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
font: 50 15pt &quot;Courier&quot;;
pading:20px;
}</string>
    </property>
    <property name="placeholderText">
     <string>请输入登录账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 50 15pt &quot;Courier&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2/user.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2/key.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginPushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 16pt &quot;微软雅黑&quot;;
}

QPushButton:pressed{
padding-top:10px;
padding-left:10px;

}</string>
    </property>
    <property name="text">
     <string>登陆 &gt;</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/loging_button_bj.png);
font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="forgetPasswordPushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>forgetPasswordPushButton</zorder>
   <zorder>usernameLineEdit</zorder>
   <zorder>passwordLineEdit</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>loginPushButton</zorder>
   <zorder>label</zorder>
   <zorder>checkButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="serverAction"/>
    <addaction name="registerAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="registerAction">
   <property name="icon">
    <iconset>
     <normaloff>image/2/user.png</normaloff>
     <normalon>:/image/login/管理员.png</normalon>image/2/user.png</iconset>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="serverAction">
   <property name="icon">
    <iconset>
     <normaloff>image/2/change.png</normaloff>image/2/change.png</iconset>
   </property>
   <property name="text">
    <string>设置服务器地址</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
